lua/textcase/plugin/conversion.lua

```diff
@@ -1,6 +1,9 @@
 local utils = require("textcase.shared.utils")
 local lsp = vim.lsp
 
+-- local flag_buf_request_all = vim.fn.has("nvim-0.10") == 1
```

Comment:
Will be enabled in a follow up PR

Reply:
`buf_request_all` has been added 2 years ago to Neovim: https://github.com/neovim/neovim/commit/ef5314ce5978db9fb57ceb4f243c36be62936294

So we are already able to use it already in Neovim v0.8 I guess.

Reply:
Which means we need to think about what versions of Neovim to support. Probably v0.8 is enough when 0.10 is just around the corner and then we don't need a flag at all.